docs/generation/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14-bin.zip
```

Comment:
Release notes recommend updating the wrapper as well: https://docs.gradle.org/8.14/release-notes.html#upgrade-instructions
Could you please follow-up with running that command for both main and docs subproject?